src/cryptoadvance/specter/templates/base.jinja

```diff
@@ -10,6 +10,34 @@
 			img[src="{{ url_for('static', filename='img/devices/elementscore_icon.svg') }}"] {
 				transform: scale(2.2);
 			}
+			{% if session["language_code"] in config["RTL_LANGUAGES"] %}
```

Comment:
I expect that to be an idiom which might come up hundreds of times in the code:
```
session["language_code"] in config["RTL_LANGUAGES"]
```
Can we make that much shorter? something like `session["l10n_rtl"] `